core/src/main/resources/jenkins/install/SetupWizard/index.jelly

```diff
@@ -2,7 +2,6 @@
 <l:html norefresh="true" title="${it.displayName}" xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:main-panel>
     <div class="plugin-setup-wizard-container"></div>
-    <script src="${resURL}/${j.installWizardPath}.js" type="text/javascript"/>
-    <link rel="stylesheet" href="${resURL}/${j.installWizardPath}.css" type="text/css" />
+    <st:adjunct includes="${j.installWizardPath}" />
```

Comment:
IMO, `<st:adjunt />` is not needed since all views in the setup wizard are using `html.jelly` as template and this template [has included](https://github.com/recena/jenkins/blob/JENKINS-34244/core/src/main/resources/lib/layout/html.jelly#L159-159) the CSS and JS files.